operator/v1/types_csi_cluster_driver.go

```diff
@@ -40,14 +40,15 @@ type CSIDriverName string
 // If you are adding a new driver name here, ensure that kubebuilder:validation:Enum is updated above
 // and 0000_90_cluster_csi_driver_01_config.crd.yaml-merge-patch file is also updated with new driver name.
 const (
-	AWSEBSCSIDriver    CSIDriverName = "ebs.csi.aws.com"
-	AzureDiskCSIDriver CSIDriverName = "disk.csi.azure.com"
-	GCPPDCSIDriver     CSIDriverName = "pd.csi.storage.gke.io"
-	CinderCSIDriver    CSIDriverName = "cinder.csi.openstack.org"
-	VSphereCSIDriver   CSIDriverName = "csi.vsphere.vmware.com"
-	ManilaCSIDriver    CSIDriverName = "manila.csi.openstack.org"
-	OvirtCSIDriver     CSIDriverName = "csi.ovirt.org"
-	KubevirtCSIDriver  CSIDriverName = "csi.kubevirt.io"
+	AWSEBSCSIDriver                  CSIDriverName = "ebs.csi.aws.com"
```

Comment:
@jsafrane - per the comment on line 41, I see you removed use of the `kubebuilder:validation:Enum` directive for the `CSIDriverName` type with https://github.com/openshift/api/pull/718

Hence, the manual update of both 000_90... yaml files

Now, I did try in my local copy of this branch adding that directive back, and after undoing the change to the crd yam (but not the patch yaml) a `make update` did add my new driver to the crd yaml

I take your https://github.com/openshift/api/pull/718#discussion_r471588189 as meaning the removal of the kubebuilder comment for CSIDriverName was intentional.

Anyway, it seems to me if we leave things as is, the comment saying update kubebuilder:validation:Enum should be changed, since that directive is no longer specified.

Or would you like me to add it back?

Reply:
Yes, the comment is wrong, there is no `kubebuilder:validation:Enum` now. Can you please remove the comment?

If I remember correctly, a patch was the only way to add validation of `metadata.name`.

Reply:
yep I'll update the comment ... fighting a few fires this AM, but hopefully will still circle back to this today

Reply:
mention of kubebuilder Enum in the comment removed @jsafrane